daml-lf/kv-support/src/main/scala/com/daml/lf/kv/transactions/TransactionTraversal.scala

```diff
@@ -20,6 +23,27 @@ object TransactionTraversal {
   def traverseTransactionWithWitnesses(rawTx: RawTransaction)(
       f: (RawTransaction.NodeId, RawTransaction.Node, Set[Ref.Party]) => Unit
   ): Either[ConversionError, Unit] =
+    for {
+      parsedTransaction <- parseTransaction(rawTx)
+      (txVersion, nodes, initialToVisit) = parsedTransaction
```

Comment:
Nit: Do we have to split it into separate lines?

Reply:
Yes, unfortunately it's a known issue.

Reply:
Yes. See the [post by Stephen ](https://discuss.daml.com/t/pattern-always-adds-withfilter-scala-ritual/1463)for more details